Core/src/main/java/com/plotsquared/core/generator/HybridPlotWorld.java

```diff
@@ -456,8 +490,52 @@ public void addOverlayBiome(short x, short z, BiomeType id) {
         this.G_SCH_B.put(pair, id);
     }
 
+    /**
+     * Get the entities contained within the plot schematic for generation. Intended for internal use only.
+     *
+     * @since TODO
+     */
+    @AnnotationHelper.ApiDescription(info = "Internal use only. Subject to changes at any time.")
+    public @Nullable List<Entity> getPlotSchematicEntities() {
+        return schem3Entities;
+    }
+
+    /**
+     * Get the minimum point of the plot schematic for generation. Intended for internal use only.
+     *
+     * @since TODO
+     */
+    @AnnotationHelper.ApiDescription(info = "Internal use only. Subject to changes at any time.")
+    public @Nullable BlockVector3 getPlotSchematicMinPoint() {
+        return schem3MinPoint;
+    }
+
+    /**
+     * Get if post-generation population of chunks with tiles/entities is needed for this world. Not for public API use.
+     *
+     * @since TODO
+     */
+    @AnnotationHelper.ApiDescription(info = "Internal use only. Subject to changes at any time.")
+    public boolean populationNeeded() {
+        return schem1PopulationNeeded || schem2PopulationNeeded || schem3PopulationNeeded;
```

Comment:
Either we change the signature name and/or invert the return, otherwise the name is quite misleading, as the outcome is always inverted.

Reply:
The intillij warning is because the only place it's used is in a negation